src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -0,0 +1,607 @@
+import logging
+import math
+import os
+import threading
+import time
+
+import numpy
+import psutil
+from concurrent.futures._base import RUNNING, FINISHED, CANCELLED, CancelledError
+
+from odemis import model, dataio
+from odemis.acq import acqmng
+from odemis.acq import stitching
+from odemis.acq.align.autofocus import MeasureOpticalFocus, AutoFocus, MTD_EXHAUSTIVE
+from odemis.acq.stream import Stream, SEMStream, CameraStream, RepetitionStream, EMStream, ARStream, \
+    SpectrumStream, FluoStream, MultipleDetectorStream, util, executeAsyncTask, \
+    CLStream
+from odemis.util import dataio as udataio
+from odemis.util.comp import compute_scanner_fov, compute_camera_fov
+
+WEAVER_COLLAGE_REVERSE = 2
```

Comment:
Please import it, don't duplicate.

Reply:
The reason for adding it is that `WEAVER_COLLAGE_REVERSE` is defined in the same `__init__` file that imports the `_tiledacq` functions. What do you recommend in order to avoid circular imports?

https://github.com/delmic/odemis/pull/1455/files#diff-2e25268b4d6583980658f17ba41cc30f

Reply:
I see... yes, this kind of issue creeps in from time to time. Definitely, you must use an import here.

In general, I feel it's a tricky part in Python. You want to have the constants and abstract classes in the root of the module, for everyone to see, so you put them in the `__init__.py` file... but then the internal modules also need them and you end-up with circular deps.

I'd suggest to move the constants to a separate private submodule. You can call it `_constants.py`, or `_interface.py` (but if you have a better name, go for it!)